src/plugins/governance/fetch-data.ts

```diff
@@ -0,0 +1,113 @@
+import got from 'got';
+import remark from 'remark';
+import gfm from 'remark-gfm';
+import toString from 'mdast-util-to-string';
+import { Node, Parent } from 'unist';
+import { Table, Link } from 'mdast';
+import * as dotenv from 'dotenv';
+dotenv.config();
+
+export interface WorkingGroup {
+  name: string;
+  description: string;
+  chair: string;
```

Comment:
Should we be forward looking and make `chair` optional?

Reply:
yeah, that makes sense.

Reply:
9600d2e